force-app/main/default/classes/BDI_DataImportService.cls

```diff
@@ -346,6 +346,36 @@ global with sharing class BDI_DataImportService {
         }
     }
 
+    /*******************************************************************************************************
+    * @description Instrument that multiple gift were successfully processed through the DataImport Batch process
+    * @param Id Apex Batch Job to find related DataImport__c records for
+    * @return void
+    */
+    public static void instrumentGiftEntry(Id apexJobId) {
+        try {
+            String createStatus = System.Label.bdiCreated;
+            Integer insertedGifts = [SELECT Count() 
+                FROM DataImport__c 
+                WHERE ApexJobId__c = :apexJobId
+                AND DonationImportStatus__c = :createstatus];
+                instrumentGiftEntry(insertedGifts);
```

Comment:
Bit of a nitpick, but can we un-tab this line? I thought it was part of the query at first glance!